jsonrpc/eth_endpoint.go

```diff
@@ -471,6 +489,12 @@ func (e *Eth) GetTransactionCount(address types.Address, number BlockNumber) (in
 
 // GetCode returns account code at given block number
 func (e *Eth) GetCode(address types.Address, number BlockNumber) (interface{}, error) {
+
+	//Set the block number to latest
+	if BlockNumber(0) == number {
+		number = LatestBlockNumber
+	}
```

Comment:
@munna0908 same as the above. we need to have a way to query the genesis block storage/balances/code since our entire PoS implementation depends on predeployed contracts.

Reply:
sure @lazartravica , will modify this accordingly

Reply:
Good catch @lazartravica 💯

Reply:
Fixed it, Commit [link](https://github.com/0xPolygon/polygon-sdk/commit/7ed9279cd30c72299f4073dc9bf906cde80e4724)